modules/shortcodes/getty.php

```diff
@@ -47,39 +82,73 @@ function jetpack_getty_enable_embeds( $site = 'wpcom' ) {
  * @return mixed
  */
 function wpcom_shortcodereverse_getty( $content ) {
-	if ( ! is_string( $content ) || false === stripos( $content, 'embed.gettyimages.com/embed' ) ) {
+	if ( ! is_string( $content ) || false === stripos( $content, '.gettyimages.com/' ) ) {
 		return $content;
 	}
 
-	$regexp = '!<iframe\s+src=[\'"](https?:)?//embed\.gettyimages\.com/embed(/|/?\?assets=)(\d+(,\d+)*)[^\'"]*?[\'"]((?:\s+\w+=[\'"][^\'"]*[\'"])*)((?:[\s\w]*))></iframe>!i';
+	$regexp = '!<iframe\s+src=[\'"](https?:)?//embed\.gettyimages\.com/embed(/|/?\?assets=)([a-z0-9_-]+(,[a-z0-9_-]+)*)[^\'"]*?[\'"]((?:\s+\w+=[\'"][^\'"]*[\'"])*)((?:[\s\w]*))></iframe>!i';
 	$regexp_ent = str_replace( '&amp;#0*58;', '&amp;#0*58;|&#0*58;', htmlspecialchars( $regexp, ENT_NOQUOTES ) );
 
-	foreach ( array( 'regexp', 'regexp_ent' ) as $reg ) {
+	/**
+	 * Context: http://wp.me/p5j4vm-1y7
+	 *
+	 * @hide-in-jetpack
```

Comment:
Looks like this made it in when it should not have?

Reply:
Thank you, removed!